### DISCUSSION TITLE
Search filter for subscribed issues/PRs

### DISCUSSION THREAD

**Discussion by SnirBroshi** [community/community#6403] [Projects and Issues]
When searching for issues/PRs in `https://github.com/search?type=issues` or a in repo such as `https://github.com/microsoft/vscode/issues`, there is no way to filter for items which I'm subscribed to, so there is no way to see the manual subscriptions.

There is the [Notifications](https://github.com/notifications/subscriptions) page, but it's not a general search - it doesn't have GitHub's normal search/filter capabilities, such as `comments:>10`, so having such a filter in the normal search gives much more control.

I suggest adding a filter such as `is:subscribed` or `subscribed:@me` to GitHub's global/repo search, which should filter for subscriptions for any reason, not just manual.

There are ways to find all non-manual subscription reasons, such as `involves:@me` / `commenter:@me` / `mentions:@me` / `author:@me`.

I'd also love `reacted:username` to search for issues/PRs where a user has reacted to any of the comments in them.

**Comment by jglick** [comment]
For comparison, in Jira I would just use the JQL clause `watcher = currentUser()`.

The notifications page is particularly unhelpful given #7144.

**Comment by MagnusWikensten** [comment]
I would also like to see an `is:subscribed` search filter. That would help a lot in the daily work for me as a codeowner but who does not have to take action on all PR's, hence unsubscribing these. Then I would be able to filter these unsubscribed ones out in my PR list, using e.g. `-is:subscribed`.

**Comment by Mart-Bogdan** [comment]
I would like intuitive way to search issues I am subscribed to, or ones that I had commented.
It's mostly if somebody other created a BUG and I also experiencing same problem. 

I went to "Issues" button on top menu, but it only have search preset for Created/Assigned/Mentioned.

If I made comment, but there were not replies, or replier didn't bothered to add a @mention -- Issue won't show there.


![image](https://user-images.githubusercontent.com/775038/210383092-b6178956-75c2-458e-b686-ef14dc0cf710.png)

There are some workarounds.
1. Use  [`Notifications/Subscriptions`](https://github.com/notifications/subscriptions), which is quite hard to get from UI with lots of clicks and scrolling. And it don't have search fucntionality, only sort. Which is at least somehow useful.
2. Use manual filters, like `involves:` or more granaular ones, as: `author:`, `assignee:`, `mentions:` and `commenter:`, but it's crumble some to  add this filters each time, especcialy without autocomplete on Githab search box. I've added this filters for my bookmarks bar now.

There were issues about this on old github's tracker:
1. https://github.com/isaacs/github/issues/283
2. https://github.com/isaacs/github/issues/1589

But I don't consider it (documentation [View subscriptions](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/viewing-your-subscriptions)) done .

Such easy thing as add new button "Involves" to Issues page near filter would solve this and improve UX.

**Reply by FranklinYu** [reply]
The first workaround doesn’t work for me because of #68096. For the second workaround, I don’t think the `involves:` qualifier covers subscription.

My use case is that I would like a list of issues I subscribed under a specific repository. Another workaround is to actually comment in all those issues, so that I can use the `commenter:` qualifier. Drawback is that this would spam all subscriber.
